### PR TITLE
[Enhancement] Making response showmodules view

### PR DESCRIPTION
**Pull request by AreYouIvan**
Hello team, I made some changes to turn responsive "showmodules", these changes were just in the CSS.

[Screencast from 07-09-2022 05:01:15 PM.webm](https://user-images.githubusercontent.com/73669701/178126024-434a94a0-0933-40e3-9201-68ea0b8d89c8.webm)

### CODE REVIEW

File: src/pages/showmodules/index.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+;import React from "react";
```
Comment (LilianaRestrepoTorres) [inline src/pages/showmodules/index.tsx:1]: I guess it's a typo, isn't it? @AreYouIvan

File: src/components/CardItem/index.tsx [hunk 2]
```diff
@@ -1,49 +1,67 @@
...
+          clipRule="evenodd"
+        />
+      </svg>
+      <div className="w-auto">
+        <h2 className="ml-4 text-lg font-medium leading-6 text-gray-900">
+          {item?.nickname}
+          {item?.name}
+          {item?.title?.title}
+        </h2>
+        <dd className="mt-2 ml-4 text-base text-gray-500 line-clamp-3 ">
```
Comment (LilianaRestrepoTorres) [inline src/components/CardItem/index.tsx:60]: If we still using dd tag, should we need to use dt as well? The previous one was replaced by div. @AreYouIvan

**Review by LilianaRestrepoTorres — APPROVED** [review]
Good improvement.

**Review by NousCode — COMMENTED** [review]
Nice job Ivan :+1:

**Review by Hai5edfm — APPROVED** [review]

**Pull request merged**